test/unit/pr_checker.test.js

```diff
@@ -300,4 +302,92 @@ describe('PRChecker', () => {
       assert.deepStrictEqual(logger.logs, expectedLogs);
     });
   });
+
+  describe('checkCommitAfterReview', () => {
+    let logger = null;
+
+    beforeEach(() => {
+      logger = new TestLogger();
+    });
+
+    afterEach(() => {
+      logger.clear();
```

Comment:
If we are going to create a new logger for every tests case we don't need to clean them up.